src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -24,8 +24,8 @@ struct ReduceAxesParam : public dmlc::Parameter<ReduceAxesParam> {
     DMLC_DECLARE_FIELD(axis).set_default(TShape())
       .describe("The axis or axes along which to perform the reduction. "
                 "The default, `axis=()`, will compute over all elements into a "
-                "scalar array with shape `(1,)`.\n\nIf axis is int, "
-                "a reduction is performed on a particular axis.\n\n"
+                "scalar array with shape `(1,)`. If axis is int, "
```

Comment:
The reason there is a `\n\n` is because we want the following format:

The axis ...
- If axis is int, ...
- If axis is a tuple of xxx
 
You can either use `R"code( xxx )code"` here to reserve this format, or move the if items to the description of the function.